### DISCUSSION TITLE
How to disable auto highlighting in codeBlockLowlight?

### DISCUSSION THREAD

**Discussion by cadars** [ueberdosis/tiptap#2121] [General Questions] [answered]
I would like to disable the auto highlighting, unless a language is explicitly specified with ` ```lang`.
Using `plaintext` as default would work too.

This is my setup, I'm confused about what syntax to use to disable [lowlight.HighlightAuto()](https://github.com/wooorm/lowlight#lowlighthighlightautovalue-options)
```js
const editor = useEditor({
  extensions: [
    StarterKit.configure({
      codeBlock: false,
    }),
    CodeBlockLowlight.configure({
      lowlight,
    }),
  ],
});
```
Thanks for your time!

**Comment by philippkuehn** [comment, marked as answer]
I’ve added a `defaultLanguage` option to the `CodeBlockLowlight` extension.

```js
CodeBlockLowlight.configure({
  defaultLanguage: 'plaintext',
})
```

**Reply by cadars** [reply]
That’s perfect! Thanks!
